src/workloads/sharding/BulkWriteBatchedUpdateOneWithoutShardKeyWithId.yml

```diff
@@ -0,0 +1,137 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/sharding"
+Description: |
+  Runs the batched updateOne writes of type WithoutShardKeyWithId with a batch size of 1000 using 
```

Comment:
'Runs the **bulk** updateOne writes ...'

Reply:
Hmm so it actually is batched in this case too but done differently (using bulkWrite command)

Reply:
Maybe 'Runs the batched bulk updateOne writes ...' then ?

Reply:
Done.